rotkehlchen/history/events.py

```diff
@@ -37,6 +37,7 @@
 # adex staking
 # aave lending
 # eth2
+# liquity
```

Comment:
Shouldn't you also increase the number below to 12?

Reply:
This I really thought and read several times the code. It seems like before there was 11 but there is a step where you do both. In fact I changed this to 12 and the tests were failing because the percentages didn't match

Reply:
What test failed? Since you are adding a new step and calling `self._increase_progress(step, total_steps)` the number has to increase.

Reply:
This one `rotkehlchen/tests/api/test_history.py::test_query_history_timerange` maybe there is something that I'm missing here